server/core/src/main/java/org/infinispan/server/core/admin/embeddedserver/CacheUpdateIndexSchemaTask.java

```diff
@@ -25,7 +27,14 @@
  * @since 14.0
  */
 public class CacheUpdateIndexSchemaTask extends AdminServerTask<Void> {
-   private static final Set<String> PARAMETERS = Collections.singleton("name");
+   private static final Set<String> PARAMETERS;
```

Comment:
nitpicking: could be simplified:
```suggestion
   private static final Set<String> PARAMETERS = Set.of("name", "indexed-entities");
```

Reply:
Good suggestion. Going to apply it!